rust/pkg/cardano_serialization_lib.js.flow

```diff
@@ -5340,6 +5321,11 @@ declare export class TransactionBuilder {
    */
   min_fee(): BigNum;
 }
+/**
+ */
+declare export class TransactionBuilderConfig {
+  free(): void;
+}
```

Comment:
@ozgrakkurt , using `derive(Builder)` works great from Rust but unfortunately it becomes problematic when using it from JS, everything related to `TransactionBuilderConfigBuilder` and the setter methods is just non-existing in the generated flow config.

Plz investigate if there's any way to marry the Builder deriver with the flowgen to have the proper types generated. Otherwise we will have only two options: write the builder manually and generate the types or generate the builder but write the types manually, and in that case the second option is really not great ☹️

Reply:
did this in the new commit